src/core/init.js

```diff
@@ -66,17 +70,48 @@ module.exports = (repo, opts, callback) => {
       return doneImport(null)
     }
 
-    const importer = require('ipfs-data-importing')
+    const Importer = require('ipfs-unixfs-engine').importer
     const blocks = new BlockService(repo)
     const dag = new DagService(blocks)
 
     const initDocsPath = path.join(__dirname, '../init-files/init-docs')
 
-    importer.import(initDocsPath, dag, {
-      recursive: true
-    }, doneImport)
+    const i = new Importer(dag)
+    i.on('data', (file) => {
+    })
+
+    glob(path.join(initDocsPath, '/**/*'), (err, res) => {
+      if (err) {
+        throw err
+      }
+      const index = __dirname.lastIndexOf('/')
```

Comment:
@noffle wrote that and made the note that it is not meant to run in the browser.